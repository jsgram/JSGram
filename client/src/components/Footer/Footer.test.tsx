import { Footer } from './index';

import { shallow } from 'enzyme';
import React from 'react';

describe('Footer component:', () => {
    let renderer: any;

    beforeEach(() => {
        renderer = shallow(<Footer />);
    });

    test('render - success', () => {
        expect(renderer).toMatchSnapshot();
    });
});
