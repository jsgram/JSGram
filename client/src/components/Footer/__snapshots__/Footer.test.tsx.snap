// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component: render - success 1`] = `ShallowWrapper {}`;
